### PR TITLE
chore(cts): reuse method template

### PR DESCRIPTION
**Pull request by millotp**
## 🧭 What and Why

We can reuse the `method` template in all languages, and for all tests suites, instead of copy pasting it 4 times.

Also simplify the logic and remove unused props, to make it more similar to the requests test suite

### CODE REVIEW

File: scripts/cts/testServer/timeout.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ export function assertValidTimeouts(expectedCount: number): void {
         expect(state.duration[0] * 2).to.be.closeTo(state.duration[1], 200);
         break;
       case 'Swift':
-        expect(state.duration[0]).to.be.closeTo(state.duration[1], 500);
+        expect(state.duration[0]).to.be.closeTo(state.duration[1], 800);
```
Comment (shortcuts) [inline scripts/cts/testServer/timeout.ts:32]:
```suggestion
        expect(state.duration[0]).to.be.closeTo(state.duration[1], 9000);
```

let's preshot

File: website/docs/contributing/testing/common-test-suite.md [hunk 2]
```diff
@@ -140,7 +140,7 @@ When writing your template, here is a list of variables accessible from `mustach
           "method": "the method to call on the API Client",
           "testName": "the descriptive name test (default to `method`)",
           "testIndex": "the index of the test to avoid duplicate function name",
-          "hasParameters": "true if the method has parameters, useful for `GET` requests",
+          "hasParams": "true if the method has parameters, useful for `GET` requests",
```
Comment (shortcuts) [inline website/docs/contributing/testing/common-test-suite.md:143]: makes me think we should use a json schema here instead of json

**Review by shortcuts — APPROVED** [review]
soooooooooo clean!!!

File: website/docs/contributing/testing/common-test-suite.md (hunk 2 above)
Comment (millotp) [reply website/docs/contributing/testing/common-test-suite.md:143]: yeah I can give it a try in another PR

**Pull request merged**